app/assets/stylesheets/partials/_forms.css.scss

```diff
@@ -5,3 +5,4 @@
 .date-fieldset {
   margin: 0 0 30px;
 }
+
```

Comment:
I guess this got here accidentally?

Reply:
I did fiddle with some styles - can't remember why now! I'll clean this up!